apis/nucleus/src/components/Cell.jsx

```diff
@@ -384,7 +389,12 @@ const Cell = forwardRef(
     };
 
     useEffect(() => {
-      if (initialError || !appLayout || !layout) {
+      if (initialError) {
+        // To resolve the Viz promise for missing types
+        onMount();
```

Comment:
There is no coverage for this line. If this is suppose to fix the issue then consider adding coverage for it.